presto-kudu/pom.xml

```diff
@@ -23,6 +23,86 @@
             <version>${kudu.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
```

Comment:
@arhimondr : Is including `hadoop-common` a good practice here? Or should we use Presto's shaded hadoop ?

Reply:
now I use com.facebook.presto.hadoop:hadoop-apache2 instead, Excellent advice!
https://github.com/prestodb/presto/pull/14002

Reply:
> @arhimondr : Is including `hadoop-common` a good practice here? Or should we use Presto's shaded hadoop ?
now I use com.facebook.presto.hadoop:hadoop-apache2 instead, Excellent advice!
https://github.com/prestodb/presto/pull/14002